### ISSUE TITLE
is not working with EXPO 52

### ISSUE THREAD

**Issue by yestoall** [gre/react-native-view-shot#550]
it was workint with expo 51 but it doesn't seems to work with expo 52
any ideas?

**Comment by yestoall** [comment]
i found a solution changing snapshotContentContainer: true to snapshotContentContainer: false

now its working as usual

**Comment by Tsabary** [comment]
Yup same can' seem to make it work, I keep getting "Failed to snapshot view tag 118" and I had it working before.

Adding snapshotContentContainer: false to he options doesn't seem to change it for me

**Comment by jimmyrogue** [comment]
Same, react-native-view-shot 4.0.2 work for ios  but not work for android
I just follow the tutorial https://docs.expo.dev/tutorial/screenshot/ and try to make a app with expo

**Comment by kunchit** [comment]
me too error when i update expo 52

**Comment by wasupak** [comment]
Me too, back to 4.0.0 or 3.8.0 didn't help, maybe problem with expo 52

**Comment by Camiloadh99** [comment]
I have the same issue, when i try to use the useRef.current.capture() send me an error only on android before i update to expo 52

**Comment by javalen** [comment]
> I have the same issue, when i try to use the useRef.current.capture() send me an error only on android before i update to expo 52

I rolled back to 51 and it's working. Hope we can get a fix soon because exp-go updates to 52 automatically

**Comment by Beowulf1220** [comment]
I have the same issue on Android, any solution?

**Comment by torresjr94dev** [comment]
I have the same issue on Android, but I tried on iOS and it works perfect. Adding snapshotContentContainer: false to he options doesn't seem to change it for me

**Comment by theprantadutta** [comment]
+1

**Comment by lecion** [comment]
I have the same issue

**Comment by ronix99** [comment]
I have the same issue, Expo 52

**Comment by eeebuba** [comment]
For me, i only get this error when using Expo Go. When running a development build everything works as expected.

**Comment by globemediaofficial** [comment]
> I only get this error when using Expo Go. When running a development build everything works as expected.

Can confirm that the error only occurs in Expo Go on Android. captureRef works as expected in a development or production build.

**Comment by dovidweisz** [comment]
It didn't work for me when I built it to an apk

**Comment by meathill** [comment]
By far, `captureRef` was broken, but `<ViewShot>` still works on iOS 18.1.

You guys may try: https://github.com/gre/react-native-view-shot?tab=readme-ov-file#high-level-api

**Comment by msanhueza-yont** [comment]
For RN 0.76.5, It worked for me:
- Updating the ndkVersion to “27.2.12479018”
- Update the package to version 4.0.3

**Comment by gre** [comment]
This was confirmed working with v4.0.3+ in development builds (native modules don't work in Expo Go). `v5@alpha` further improves compatibility. Can you try `npm install react-native-view-shot@alpha` and let us know if this is resolved? Stable v5 coming soon.

**Comment by gre** [comment]
v5.0.0 ships full New Architecture (Fabric + TurboModules) support across iOS and Android, plus fabric.interop wiring on Android. Please try upgrading; if the issue persists, please open a fresh bug report with logs. Closing as resolved.

Release notes: https://github.com/gre/react-native-view-shot/releases/tag/v5.0.0
